packages/react/error-boundary/src/ErrorBoundary.en.md

```diff
@@ -0,0 +1,37 @@
+# ErrorBoundary
+
+- see https://jbee.io/react/error-declarative-handling-1/ Declaratively handle loading and error conditions
+- see https://toss.im/slash-21/sessions/3-1 Suspense & error handling presentation in Slash 21
```

Comment:
How about removing them from English documentation? Because I think they are absolutely Korean specific content.

Reply:
[fix: remove korean specific content](https://github.com/toss/slash/pull/102/commits/58e1bebcd2673be8a190045812e48f0aca61a07a)
[58e1beb](https://github.com/toss/slash/pull/102/commits/58e1bebcd2673be8a190045812e48f0aca61a07a)

I fixed but There are some Koreans who read the documents in English because they are comfortable with English, so is there a good way to provide them this references

Reply:
I think people who can speak only English are more than them.